src/Surfnet/ServiceProviderDashboard/Domain/Entity/EntityDiff.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * Copyright 2022 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Domain\Entity;
+
+class EntityDiff
+{
+    private $diff;
+
+    public function __construct($data, $compareTo)
```

Comment:
Parameter typing is missing in the constructor

Reply:
Thanks, give me some of my own ℞ :mechanical_arm: